Sources/DataStructures/Bimap.swift

```diff
@@ -220,11 +220,15 @@ extension Bimap: ExpressibleByDictionaryLiteral {
 
 extension Bimap {
     
-    public static func compose <Other: Hashable> (lhs: Bimap, rhs: Bimap<Value, Other>) -> Bimap<Key, Other> {
+    public static func compose <Other: Hashable> (_ lhs: Bimap, _ rhs: Bimap<Value,Other>) -> Bimap<Key,Other> {
```

Comment:
You could also consider making this an instance method, like:

```Swift
extension Bimap {
    public func compose <O> (with other: Bimap<Value,O>) -> Bimap<Key,O> where O: Hashable {
        ...
    }
}
```

Then in the `static *`:

```Swift
static func * <...> (...) -> ... { return lhs.compose(with: rhs) }
```